libs/designer/src/lib/ui/configuretemplate/panels/configureWorkflowsPanel/usePanelTabs.tsx

```diff
@@ -29,23 +29,39 @@ export const useConfigureWorkflowPanelTabs = ({
     runValidation: state.tab.runValidation,
   }));
 
-  const hasError = false; // Placeholder for actual error state
-
   const [selectedWorkflowsList, setSelectedWorkflowsList] =
     useFunctionalState<Record<string, Partial<WorkflowTemplateData>>>(workflowsInTemplate);
 
+  const currentSelectedWorkflowsList = selectedWorkflowsList();
+  const duplicateIds = useMemo(() => {
+    const seen = new Set<string>();
```

Comment:
WHy do we have duplicateids

Reply:
With your current changes we should not have duplicateids because we are always putting unique name now